.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,18 +1,29 @@
-This PR adds a fix for the leaking coffee machine.
+This PR adds/fixes ...
 
-**How to test**:
-1. install on stage of the coffee machine: `bash servers/resources/coffee.selecta.se/update-cg-stage.sh fix-water-leak`
-1. activate stage: `us`
-1. run following command: `cg get coffee`
+### How to prepare for test
+- [ ] ssh to hasta (depending on type of change)
+- [ ] install on stage:
+`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-trailblazer-stage.sh [THIS-BRANCH-NAME]`
+- [ ] ssh to clinical-db (depending on type of change)
+- [ ] install on stage:
+`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-stage.sh [THIS-BRANCH-NAME]`
```

Comment:
```suggestion
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-ui-stage.sh [THIS-BRANCH-NAME]`
```

Reply:
Do we need both front-end and back-end on clinical-db? Any thoughts @patrikgrenfeldt

Reply:
There is only one update-script on clinical-db, it installs both frontend and backend so I think it should be sufficient with one